generators/liquibase-changelogs/incremental-liquibase.spec.mts

```diff
@@ -86,6 +86,12 @@ relationship OneToOne {
 One to Another,
 }`;
 
+const jdlApplicationWithEntitiesAndRelationshipsWithOnHandlers = `
+${jdlApplicationWithEntities}
+relationship ManyToOne {
+One to @OnDelete("CASCADE") @OnUpdate("SET NULL") Another,
```

Comment:
```suggestion
One to @LiquibaseOnDelete("CASCADE") @LiquibaseOnUpdate("SET NULL") Another,
```

Reply:
As discussed, I'd keep it technology neutral.